packages/dpkg/src-statoverride-main.c.patch

```diff
@@ -1,14 +1,16 @@
-diff -u -r ./src/statoverride/main.c ./main.c
---- ./src/statoverride/main.c	2022-05-01 09:51:32.298164682 -0300
-+++ ./main.c	2022-05-01 09:56:12.247431464 -0300
-@@ -185,8 +185,10 @@
+diff -u -r ../dpkg-1.21.8/src/statoverride/main.c ./src/statoverride/main.c
+--- ../dpkg-1.21.8/src/statoverride/main.c	2022-05-24 14:36:59.000000000 -0300
++++ ./src/statoverride/main.c	2022-05-30 13:51:27.330797786 -0300
+@@ -186,6 +186,12 @@
  static void
  statdb_node_apply(const char *filename, struct file_stat *filestat)
  {
 +#ifndef __ANDROID__
- 	if (chown(filename, filestat->uid, filestat->gid) < 0)
- 		ohshite(_("error setting ownership of '%.255s'"), filename);
++        if (chown(filename, filestat->uid, filestat->gid) < 0)
++                ohshite(_("error setting ownership of '%.255s'"), filename);
 +#endif
- 	if (chmod(filename, filestat->mode & ~S_IFMT))
- 		ohshite(_("error setting permissions of '%.255s'"), filename);
++        if (chmod(filename, filestat->mode & ~S_IFMT))
++                ohshite(_("error setting permissions of '%.255s'"), filename);
```

Comment:
Why not just remove this patch?

Reply:
Fear it may do something. I don't particularly understand it-
Only took it from the previous update and adapted to the new location of the lines

Reply:
Wait. Figured out my mistake. Yep, gonna remove the patch.